src/Microsoft.DotNet.Build.Tasks/PackageFiles/FrameworkTargeting.targets

```diff
@@ -32,7 +32,7 @@
     <!-- Setting a default portable profile, although nothing should resolve from there as we want to use the pacakge refs -->
     <TargetPlatformIdentifier>Portable</TargetPlatformIdentifier>
     <TargetFrameworkIdentifier>.NETPortable</TargetFrameworkIdentifier>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
     <TargetFrameworkProfile>Profile7</TargetFrameworkProfile>
```

Comment:
When we move to v5.0 there are no profiles so you should remove TargetFrameworkProfile. Actually you should probably just set it to empty.